pkg/interfaces/contracts/pool-hooks/IMevHook.sol

```diff
@@ -72,6 +72,13 @@ interface IMevHook {
 
     /**
      * @notice Permissioned function to set the default multiplier of the priority gas price.
+     * @dev The multiplier is not validated or limited by any value and can assume any 18-decimal number. That's
+     * because the multiplier value depends on the priority gas price used by searchers in a given moment for a
+     * specific chain. However, the resulting swap fee percentage, given by `priorityGasPrice * multiplier`, is capped
+     * in the lower end by the static swap fee, and in the upper end by the maximum swap fee percentage of the vault.
```

Comment:
```suggestion
     * at the lower end by the static swap fee, and at the upper end by the maximum swap fee percentage of the vault.
```
Here and below

Reply:
Maybe also say "18-decimal floating point number" above (which would shift everything around to fit in 120 characters.